src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -99,6 +98,9 @@ public override void OnStartActivity(Activity activity, object payload)
 
                     // Starting the new activity make it the Activity.Current one.
                     newOne.Start();
+
+                    // Set IsAllDataRequested to false for the activity created by the framework to only export the sibling activity and not the framework activity
+                    activity.IsAllDataRequested = false;
```

Comment:
@cijothomas Are we exporting these in 1.0? Could technically be breaking if so? I think it's worth doing but should be called out as a change. Might already be in changelog I haven't gotten that far in the diff yet 😄

Reply:
Instrumentation haven't reached 1.0, so okay to make breaking change. However, I dont think there is any breaking change (breaking behaviour). Previously we were exporting only one activity, which is the one we create, if we create one. The framework is not exported today, if we create sibling activity ourselves